GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/UI/AboutWindowController.m

```diff
@@ -1,20 +1,51 @@
-		#import "GVFSTask.h"
 #import "AboutWindowController.h"
 
 @interface AboutWindowController ()
 
+@property (strong) ProductInfoFetcher *productInfoFetcher;
+
 @end
 
 @implementation AboutWindowController
 
-- (void)windowWillLoad
+- (id) initWithProductInfoFetcher:(ProductInfoFetcher *) productInfoFetcher
 {
-    GVFSTask *gvfsTask = [[GVFSTask alloc] init];
-    
-    self.VFSForGitVersion = [gvfsTask VFSForGitVersion];
-    self.GitVersion = [gvfsTask GitVersion];
+    if (self = [super initWithWindowNibName:@"AboutWindowController"])
+    {
+        self.productInfoFetcher = productInfoFetcher;
```

Comment:
Should not use self.property in init methods. Use ivar instead:
_productInfoFetcher = productInfoFetcher;